launcher/add_cert_to_keystore.md

```diff
@@ -0,0 +1,27 @@
+Add below to keycloak `DeploymentConfig`
```

Comment:
This isn't used, but may be required in future. 
I'd like to keep the steps that worked captured somewhere